### DISCUSSION TITLE
My plot times only increase over time

### DISCUSSION THREAD

**Discussion by blood5322** [madMAx43v3r/chia-plotter#782] [Q&A]
I've been using MadMax plotter for a week now and my plot times started out at around 45 min each. Now I'm averaging 70min each, and I can't figure out why. I'm plotting on a dell poweredge 720xd with 2 cpus each with 8core/16 threads and 64 GB ram. My temp drive is 12, 10k sas 600gb hard drives all raided together. Running on Ubuntu 20.04 I've been plotting with -r 30. Any help would be greatly appreciated, as a note I am also farming on this server while plotting. The final plots go to internal sata drives.

**Comment by alt-savage** [comment]
Data fragmentation on your SAS drives?  For those experiencing this issue on NVMEs, be sure to run fstrim on your drive before every plot.

**Reply by altendky** [reply]
Instead of trimming, you can mount with the `discard` option.

**Reply by blood5322** [reply]
I can try that, I've never used that before.

**Comment by alt-savage** [comment]
With btrfs I've had best performance when mounting  with discard=async 

Trim before every plot with fstrim

For optimal full drive drive trim, re-mount your nvme right before the trim. (umount and then mount). 
This allows every single block in SSD to be trimmed, restoring full SSD performance.
